packages/legend-query/src/components/QueryBuilderMilestoneEditor.tsx

```diff
@@ -206,20 +204,20 @@ const BiTemporalMilestoneEditor = observer(
       <>
         <div className="panel__content__form__section">
           <div className="panel__content__form__section__header__label">
-            Business Date
+            Processing Date
           </div>
           <MilestoningParameterEditor
-            key="BusinessDate"
+            key="ProcessingDate"
             queryBuilderState={queryBuilderState}
             index={0}
           />
         </div>
         <div className="panel__content__form__section">
           <div className="panel__content__form__section__header__label">
-            Processing Date
+            Business Date
           </div>
           <MilestoningParameterEditor
-            key="ProcessingDate"
+            key="BusinessDate"
```

Comment:
Sorry if I asked before, but why do we need to have this prop? We're not using a `list` anywhere

Reply:
We are using `MilestoningParameterEditor` twice for a `bitemporal` class, so adding a key